octopus/modules/epmc/client.py

```diff
@@ -20,6 +20,17 @@ def quote(s, **kwargs):
         return None
 
 
+def check_epmc_version(resp_json):
+    try:
+        received_ver = resp_json['version']
+        configured_ver = app.config.get("EPMC_TARGET_VERSION")
+        if received_ver != configured_ver:
+            app.logger.warn("Mismatching EPMC API version; recommend checking for changes. Expected '{0}' Found '{1}'".format(configured_ver, received_ver))
+    except KeyError:
+        # The json doesn't have a version key, that's not the end of the world.
+        pass
```

Comment:
Hmm. I'd log this, otherwise you can just fail to tell if the version's changed for some other reason. So "Couldn't check EPMC API version, did not find "version" key in EPMC response. Proceed with caution as the EPMC API may have changed."

Reply:
Good point, I'll add that warning.